app/views/Data/Workspace/WorkTable/index.tsx

```diff
@@ -1,11 +1,17 @@
+import { WorkTableQuery } from '#gql/graphql';
```

Comment:
It's better to follow the import ordering in the project. I think it's the react library imports in the beginning, and other libraries and the local ones. right @samshara ?

Reply:
@bewakes yes. @shreeyash07 we should follow the import order.